drivers/i2c/i2c_ll_stm32.c

```diff
@@ -20,6 +21,23 @@ LOG_MODULE_REGISTER(i2c_ll_stm32);
 
 #include "i2c-priv.h"
 
+#if defined(CONFIG_SOC_SERIES_STM32F3X) || defined(CONFIG_SOC_SERIES_STM32F0X)
```

Comment:
This changes are not related to code generation. Remove from there

Reply:
In these drivers I added changes whenever CI failed. As already mentioned it is a proof of concept. I do not claim that every driver is right but I want CI to pass. I can not demonstrate the viability of Codegen without CI to pass.